docs/source/index.rst

```diff
@@ -46,7 +47,7 @@ be used anywhere where high-throughput and robust messaging are needed.
 Features
 ++++++++
 
-* Support for 1000s of messages per second
+* Support for `1000s of messages per second <performance>`__
```

Comment:
This link does not work. I am not sure how to fix it keeping this link-referecing format, but I am sure you could use ``:ref:`1000s of messages per second <performance>` `` and then add `.. _performance::` before the "Performance" section header in `docs/source/performance.rst`.

Reply:
Whoops, missed a trailing underscore in `performance`.  Should be good now.

Reply:
Uhm, I don't see any changes. Are you sure you have pushed and to the correct branch?

Reply:
Doh.  Was wrong branch.  Good now.